<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kristall Browser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/kristall.svg</normaloff>:/icons/kristall.svg</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="BrowserTabWidget" name="browser_tabs">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="outline_window">
   <property name="windowIcon">
    <iconset theme="table-of-contents">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="windowTitle">
    <string>Document &amp;Outline</string>
   </property>
   <property name="_shortcut" stdset="0">
    <string>Ctrl+M</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="outline_view">
       <property name="autoExpandDelay">
        <number>0</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="bookmarks_window">
   <property name="windowIcon">
    <iconset theme="bookmarks">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="windowTitle">
    <string>&amp;Favourites</string>
   </property>
   <property name="_shortcut" stdset="0">
    <string>Ctrl+B</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTreeView" name="favourites_view">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="history_window">
   <property name="windowIcon">
    <iconset theme="history">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="windowTitle">
    <string>&amp;History</string>
   </property>
   <property name="_shortcut" stdset="0">
    <string>Ctrl+H</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListView" name="history_view"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionNew_window"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionManage_Certificates"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Window"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionChangelog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionShow_document_source"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionGo_to_home"/>
    <addaction name="actionBackward"/>
    <addaction name="actionForward"/>
    <addaction name="actionRoot"/>
    <addaction name="actionParent"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_to_favourites"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="settings">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Back</string>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>For&amp;ward</string>
   </property>
   <property name="toolTip">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionRoot">
   <property name="icon">
    <iconset theme="go-top">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>R&amp;oot</string>
   </property>
   <property name="toolTip">
    <string>Go to the root directory (/)</string>
   </property>
   <property name="shortcut">
    <string>Alt+/</string>
   </property>
  </action>
  <action name="actionParent">
   <property name="icon">
    <iconset theme="go-up">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Parent</string>
   </property>
   <property name="toolTip">
    <string>Go to the parent directory</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/common/qt.svg</normaloff>:/icons/common/qt.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd_to_favourites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/light/actions/favourite-on.svg</normaloff>:/icons/light/actions/favourite-on.svg</iconset>
   </property>
   <property name="text">
    <string>Add to &amp;favourites</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionGo_to_home">
   <property name="icon">
    <iconset theme="go-home">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;home</string>
   </property>
   <property name="shortcut">
    <string>Alt+Home</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset theme="help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="icon">
    <iconset theme="changelog">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Change&amp;log</string>
   </property>
  </action>
  <action name="actionManage_Certificates">
   <property name="icon">
    <iconset theme="view-certificate">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage Certificates…</string>
   </property>
  </action>
  <action name="actionShow_document_source">
   <property name="text">
    <string>View document &amp;source</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New &amp;window</string>
   </property>
   <property name="toolTip">
    <string>Opens a new window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose_Window">
   <property name="text">
    <string>Close Win&amp;dow</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowserTabWidget</class>
   <extends>QTabWidget</extends>
   <header>widgets/browsertabwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
